### ISSUE TITLE
requires oauth 0.8.0

### ISSUE THREAD

**Issue by raarts** [ueberauth/ueberauth_facebook#33]
Other ueberauth modules require 0.8. In the mean time oauth2 0.9.0 was released which fixed a bunch of compile warnings.

**Comment by njwest** [comment]
Experiencing same issue: 
```
Failed to use "oauth2" (version 0.9.1) because
  ueberauth_facebook (version 0.6.0) requires ~> 0.8.0
  ueberauth_github (version 0.4.1) requires ~> 0.8
  mix.lock specifies 0.9.1

** (Mix) Hex dependency resolution failed, relax the version requirements of your dependencies or unlock them (by using mix deps.update or mix deps.unlock). If you are unable to resolve the conflicts you can try overriding with {:dependency, "~> 1.0", override: true}
```

**Comment by doomspork** [comment]
@raarts I've released 0.7.0 which has bumped the dependencies 👍
